fe/fe-core/src/test/java/com/starrocks/catalog/CreateTableTest.java

```diff
@@ -1760,6 +1760,73 @@ public void testCreatePartitionByExprTable() {
                                 "\"storage_format\" = \"DEFAULT\"\n" +
                                 ");"
                 ));
+
+        ExceptionChecker.expectThrowsWithMsg(AnalysisException.class,
+                "Getting analyzing error from line 5, column 24 to line 5, column 41. Detail message: " +
```

Comment:
This line can be removed because it may change dynamically.

Reply:
If don't do this, is there any way to test that it really checks out the wrong table statement?

Reply:
OK, I'll fix it.